calculator.js

```diff
@@ -1,5 +1,69 @@
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
-}
+const exampleInput = {
+  num1: 8,
+  num2: 0.9,
+  operation: 'MULtiplY',
+};
+
+const operations = [
+  'add', '+', 
+  'subtract', '-', 
+  'multiply', '*', 
+  'divide', '/'
+];
+
+
+const isOperationIncluded = (operation) => {
+  return operations.includes(operation);
+};
```

Comment:
Since this function is only one line I personally would probably have just used `operations.includes(operation)` in place of it.